src/geometry/euclidean/circumcenter.lean

```diff
@@ -335,6 +335,79 @@ begin
   exact h.2
 end
 
+/-- The circumradius is non-negative. -/
+lemma circumradius_nonneg {n : ℕ} (s : simplex ℝ P n) : 0 ≤ s.circumradius :=
+s.dist_circumcenter_eq_circumradius 0 ▸ dist_nonneg
+
+/-- The circumradius of a simplex with at least two points is
```

Comment:
At least two points, or positive dimension? It's morally the same, but I think the latter is closer to the statement.

Reply:
This is related to a question of terminology in how we refer to `simplex.points`. That name has the obvious advantage of being shorter than `simplex.vertices`, but do we then refer to all the values in the range of `s.points` as being the points, vertices or both of `s`? (And then we don't talk about the dimension of a simplex at all at present, although it's referred to as an n-simplex and `affine_span_eq_top_of_affine_independent_of_card_eq_findim_add_one` gives the dimension of (the direction of) the affine span (of the range of `s.points`).)

Reply:
Ok... if I read "points" in the docstring as "vertices" (which I did in the lean code without realising), then it makes sense again.